loaders.rb

```diff
@@ -0,0 +1,31 @@
+class Loader
+  def load_person_from_file(persons)
+    if File.exist?('./json/people.json')
+      JSON.parse(File.read('./json/people.json')).each do |person|
+        persons << person
+      end
+    else
+      persons = []
+    end
+  end
```

Comment:
- [ ] I think your method couldn't be saved books because after creating the book it doesn't create a file or saved them in JSON it is just created the book without adding it to the book.json file so kindly make a method so it can be saved after creation

![image](https://user-images.githubusercontent.com/58553711/201101732-601d68dd-eecc-4580-a0ec-e534da314faa.png)

Reply:
Hello @Reem-lab ,
Thank you for taking out to review our project. As regards your comment above, we aren't saving at run time rather we are saving on exit.